server/game/cards/08-MotC/TheSpearRushesForth.js

```diff
@@ -0,0 +1,27 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { CardTypes, Players } = require('../../Constants');
+
+class TheSpearRushesForth extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Bow a participating character',
+            condition: () => this.game.isDuringConflict(),
+            cost: AbilityDsl.costs.dishonor({
+                cardType: CardTypes.Character,
+                controller: Players.Self,
+                cardCondition: card => card.isHonored && card.isParticipating()
```

Comment:
this is fine - you only need `cardCondition`, because costs only allow you to pick your own characters by default, and the game knows that only characters can be dishonored

Reply:
Makes sense. Thanks !